server-common/src/main/java/org/apache/gravitino/server/authorization/jcasbin/JcasbinAuthorizer.java

```diff
@@ -216,6 +216,12 @@ public boolean isMetalakeUser(String metalake) {
     if (StringUtils.isBlank(currentUserName)) {
       return false;
     }
+
+    // Service admins are considered users of all metalakes
+    if (isServiceAdmin()) {
```

Comment:
Do we still need this?

Reply:
I think so.
Service admins need to access any metalake for administrative operations without being explicitly added as users ?

Will it work without this check